include/dav1d/picture.rs

```diff
@@ -479,7 +479,9 @@ impl Rav1dPicAllocator {
         pic.data = Some(Arc::new(Rav1dPictureData {
             // SAFETY: `MaybeUninit<u8>` should be safe for anything.
             data: array::from_fn(|i| {
-                let ptr = data[i].unwrap().cast::<AlignedPixelChunk>();
+                let ptr = data[i]
+                    .unwrap_or_else(NonNull::dangling)
```

Comment:
Looks like `NonNull::dangling` may not have the same alignment as `AlignedPixelChunk` and `Rav1dPictureDataComponent::new` fails a couple lines below.

Reply:
Uhh, okay. I need to do the cast first, then it should work.

Reply:
What tests are you running for this?

Reply:
Running a modified argon streams script that calls `dav1d` with `--negstride`

Reply:
Fixed it now, I think.  Let me know if it worked.  I tested it on a few things with `--negstride` and it seemed to work.

Reply:
Works for me